Lib/statistics.py

```diff
@@ -267,15 +267,15 @@ def _convert(value, T):
 
 
 def _find_lteq(a, x):
-    'Locate the leftmost value exactly equal to x'
+    """Locate the leftmost value exactly equal to x"""
     i = bisect_left(a, x)
     if i != len(a) and a[i] == x:
         return i
     raise ValueError
 
 
 def _find_rteq(a, l, x):
-    'Locate the rightmost value exactly equal to x'
+    """Locate the rightmost value exactly equal to x"""
```

Comment:
```suggestion
    'Locate the rightmost value exactly equal to x'
```

Reply:
Will revert this change in the next commit.